testnets/qwoyntestnet/assetlist.json

```diff
@@ -1,26 +1,46 @@
 {
-    "$schema": "../assetlist.schema.json",
-    "chain_name": "qwoyntestnet",
-    "assets": [
-      {
-        "description": "QWOYN is the native governance token for the Qwoyn Studios Platform",
-        "denom_units": [
-          {
-            "denom": "uqwoyn",
-            "exponent": 0
-          },
-          {
-            "denom": "qwoyn",
-            "exponent": 6
-          }
-        ],
-        "base": "uqwoyn",
-        "name": "Qwoyn Blockchain",
-        "display": "qwoyn",
-        "symbol": "QWOYN",
-        "logo_URIs": {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/qwoyntestnet/images/qwoyn.png"
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "qwoyn",
```

Comment:
chain_name should remain as qwoyntestnet

Reply:
Ok thats fine.

Reply:
Will this be fixed?